.cirrus.yml

```diff
@@ -0,0 +1,23 @@
+smoke_task:
+    env:
+       CIRRUS_CLONE_DEPTH: 10
+    setup_script:
+       - dnf -y install python3 git xz tcpdump nc iproute iputils gcc python3-devel qemu-kvm qemu-img
+       - python3 -m pip install $AVOCADO_SOURCE
+       - python3 -m pip install -r requirements.txt
+       - python3 setup.py develop --user
+    bootstrap_script:
+       - python3 -m avocado vt-bootstrap --vt-skip-verify-download-assets --yes-to-all
+    list_script:
+       - python3 -m avocado list
+    dry_run_script:
+       - python3 -m avocado --show all run --dry-run -- boot
+    container:
+        matrix:
+          - image: fedora:30
+          - image: fedora:29
```

Comment:
Do you expect failures between those two versions? If not than I'd simply chose the latest to not to unnecessary waste resources.

Reply:
I don't know, and that's the point... I want to be sure it doesn't.  And please, let's not talk about global warming... :)

Given that one of the goals of these checks is to make sure Avocado-VT plays well with Avocado, I would even include the one other platform we support in Avocado (RHEL 8).  But, given that we don't have all the packages needed for Avocado-VT on RHEL 8, I left it to Fedora only.

Reply:
Well, nothing is for free, which is why we should always try our best to not to overuse such services. Anyway 2 images seems fine to me, it's just that F29 and F30 are so close to each other...